### ISSUE TITLE
How to clear the contents of the terminal

### ISSUE THREAD

**Issue by csyj501** [Microsoft/vscode#23140]
- VSCode Version:
- OS Version:

Steps to Reproduce:
d:\test>cls
"cls" does not work when the contents is too long
How to clear the contents of the terminal

**Comment by csyj501** [comment]
Terminal scroll bar is displayed "CLS" doesn't work

**Comment by UthaiahBollera** [comment]
In OSX u can use **cmd+k**.

**Comment by nguyennb9** [comment]
Enter command `clear`, if you are using Mac OS, press Cmd+K.

**Comment by csyj501** [comment]
think you all,but i'm using windows!

**Comment by Tyriar** [comment]
@csyj501 ctrl+k should work on Windows

**Comment by sebbean** [comment]
command +k on mac waits for a second key "chord" 

is that just me?

**Comment by Tyriar** [comment]
@sebbean you need the terminal focused. If it doesn't work then you have keybindings overriding cmd+k

**Comment by sebbean** [comment]
```
  {
    "key": "cmd+k",
    "command": "workbench.action.terminal.clear",
    "when": "terminalFocus"
  },
```

worked for me

**Comment by ajbeaven** [comment]
This doesn't work for me on windows running VSCode version 1.17.2. The Keyboard Shortcuts page says the `Clear` command is using the default. I get the text `^K` in the terminal when I try it.

**Comment by Tyriar** [comment]
@ajbeaven I don't think Windows has a default for this, if you want ctrl+k then add this to your keybindings.json:

```js
  {
    "key": "ctrl+k",
    "command": "workbench.action.terminal.clear",
    "when": "terminalFocus"
  },
```

**Comment by ajbeaven** [comment]
@Tyriar Oh, OK. I'm just confused with the following comment you made that every seems to be upvoting for some reason.

> @csyj501 ctrl+k should work on Windows

This should be added to the default keybindings IMO.

**Comment by Tyriar** [comment]
@ajbeaven actually that guy's pretty smart, you should listen to him 😉, ctrl+k should work on Windows, it's just Linux that it's disabled on:

https://github.com/Microsoft/vscode/blob/7dd39116e7fc52f3e6ee587b3d3e8fdddf0f294c/src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts#L338-L341

If it doesn't work, you likely have a ctrl+k chord keybinding defined, if that is the case you will need to redefine the ctrl+k keybinding at the *bottom* of your keybindings.json:

```js
  {
    "key": "ctrl+k",
    "command": "workbench.action.terminal.clear",
    "when": "terminalFocus"
  },
```

**Comment by ajbeaven** [comment]
Haha, yes indeed! I'm pretty sure I don't have any custom bindings already on `ctrl+k`. If that were the case the Keyboard Shortcuts page would indicate that it was a "User" instead of "Default" like it has now. 

There are only 4 other entries in keybindings.json and none of them are `ctrl+k`. I've since added the recommended entry to the bottom and it started working. Of course, I figured that would work, but wanted to mention it here just in case it wasn't the default.
